Documentation/test-suites-overview.md

```diff
@@ -0,0 +1,240 @@
+# Mender QA test suites overview
+
+This document gives an overview of all the different tests suites that we have
+in Mender
+
+## Introduction: test levels
+
+At Mender we have different test suites that follow the Development Lifecycle,
+verifying our software at different levels through the different phases of the
+Development Lifecycle.
+
+Due to our microservices architecture, we name our test suites in a different
+way than standards defined by ISTQB (International Software Testing
+Qualifications Board) or other online references. See (ISTQB Foundation Level
+Syllabus)[https://www.istqb.org/downloads/syllabi/foundation-level-syllabus.html].
+
+Nevertheless, we follow QA best practices and principles in verifying our
+software product and its individual components at different levels.
+
+## Introduction: programming languages
+
+Except for the GUI and some client addons in C, the core components of Mender
+are written in Go.
+
+The unit-level tests are written in the same programming language as the code
+itself (usually Go), while for every other test suite we use the Python
```

Comment:
```suggestion
itself (usually Go) and the GUI code is tested using [Jest](https://jestjs.io/) and [Cypress](https://www.cypress.io), while for every other test suite we use the Python
```

Reply:
@mzedel but is this for unit tests or for e2e tests?

Reply:
Jest is for unit tests, Cypress for e2e tests